diff --git a/data/appdata/org.gnome.Todo.appdata.xml.in b/data/appdata/org.gnome.Todo.appdata.xml.in
index 1791438..62aeb22 100644
--- a/data/appdata/org.gnome.Todo.appdata.xml.in
+++ b/data/appdata/org.gnome.Todo.appdata.xml.in
@@ -24,7 +24,7 @@
       <caption>GNOME To Do with the dark theme variant</caption>
     </screenshot>
     <screenshot width="1200" height="675">
-      <image>https://git.gnome.org/browse/gnome-todo/plain/data/appdata/editing-tasklist.png</image>
+      <image>https://git.gnome.org/browse/gnome-todo/plain/data/appdata/editing-tasklists.png</image>
       <caption>Editing a tasklist with GNOME To Do</caption>
     </screenshot>
     <screenshot width="1200" height="675">
@@ -207,4 +207,5 @@
 
   <update_contact>georges.stavracas@gmail.com</update_contact>
   <translation type="gettext">gnome-todo</translation>
+  <content_rating type="oars-1.1" />
 </component>
